tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+envlist =
+    {py39, py38, py37, py36, py35, py27, pypy3, pypy2}{, -path, -sdist, -wheel}
+    type
+    docs
+isolated_build = true
+skip_missing_interpreters = true
+minversion = 3.14
+
+[testenv]
+description = run the unit tests with pytest under {basepython}
+setenv =
+    path: TEST_MODE = path
+    sdist: TEST_MODE = sdist
+    wheel: TEST_MODE = wheel
+passenv =
+    PYTEST_*
+    PIP_*
+
+skip_install = path,sdist,wheel: false
+deps =
+    path,sdist,wheel: -rrequirements-dev.txt
+extras =
+    test
+commands =
+    pytest --cov --cov-config setup.cfg \
+      --cov-report=html:{envdir}/htmlcov \
+      --cov-report=xml:{toxworkdir}/coverage.{envname}.xml \
+      tests {posargs}
+
+[testenv:type]
+description = run type check on code base
+deps =
+    mypy==0.782
+commands =
+    mypy --strict --python-version 3.8 src/build
+    mypy --strict --python-version 2.7 src/build
```

Comment:
Also, just notice now, `--strict` could be removed as we already set that in the config.

Reply:
Oh you're right fixed.